crates/bevy_internal/src/default_plugins.rs

```diff
@@ -25,59 +25,91 @@ use bevy_app::{PluginGroup, PluginGroupBuilder};
 pub struct DefaultPlugins;
 
 impl PluginGroup for DefaultPlugins {
-    fn build(&mut self, group: &mut PluginGroupBuilder) {
-        group.add(bevy_log::LogPlugin::default());
-        group.add(bevy_core::CorePlugin::default());
-        group.add(bevy_time::TimePlugin::default());
-        group.add(bevy_transform::TransformPlugin::default());
-        group.add(bevy_hierarchy::HierarchyPlugin::default());
-        group.add(bevy_diagnostic::DiagnosticsPlugin::default());
-        group.add(bevy_input::InputPlugin::default());
-        group.add(bevy_window::WindowPlugin::default());
+    fn build(self) -> PluginGroupBuilder {
+        let mut group = PluginGroupBuilder::default();
+        group
```

Comment:
```suggestion
        group = group
```
this time I checked locally 😄